iphone/TitaniumKit/TitaniumKit/Sources/API/TiUtils.m

```diff
@@ -768,6 +768,25 @@ + (UIImage *)adjustRotation:(UIImage *)image
   return imageCopy;
 }
 
++ (UIImage *)imageWithTint:(UIImage *)image tintColor:(UIColor *)tintColor
+{
+  if (![TiUtils isIOSVersionOrGreater:@"13.0"]) {
+    return [image imageWithTintColor:tintColor renderingMode:UIImageRenderingModeAlwaysOriginal];
```

Comment:
This API is available from iOS sdk 13 + only. So building on Xcode < 11 will fail. This whole function should be like -
`+ (UIImage *)imageWithTint:(UIImage *)image tintColor:(UIColor *)tintColor
{
  #if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
  if ([TiUtils isIOSVersionOrGreater:@"13.0"]) {
    return [image imageWithTintColor:tintColor renderingMode:UIImageRenderingModeAlwaysOriginal];
  } else {
#endif
    UIImage *imageNew = [image imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate];
    UIImageView *imageView = [[UIImageView alloc] initWithImage:imageNew];
    imageView.tintColor = tintColor;

    UIGraphicsBeginImageContextWithOptions(imageView.bounds.size, NO, 0.0);
    [imageView.layer renderInContext:UIGraphicsGetCurrentContext()];
    UIImage *tintedImage = UIGraphicsGetImageFromCurrentImageContext();
    UIGraphicsEndImageContext();

    return tintedImage;
  #if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
  }
  #endif
}
`